net.sourceforge.pdfchain.json

```diff
@@ -0,0 +1,84 @@
+{
+    "app-id": "net.sourceforge.pdfchain",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "40",
+    "sdk": "org.gnome.Sdk",
+    "command": "pdfchain",
+    "finish-args": [
+    	"--filesystem=xdg-documents",
+        "--filesystem=xdg-download",
+        "--filesystem=xdg-pictures",
+        "--filesystem=home:ro",
+        "--filesystem=host:ro",
```

Comment:
host include home.
 Both are IMHO excessive for a PDF editor.

Reply:
What if you want to access a PDF on a connected USB pendrive... Being ro I think gives a good balance between security and practicality. Will just remove "--filesystem=home:ro".

Reply:
```suggestion
        "--filesystem=/run/media:ro",
        "--filesystem=/media:ro",
```

Reply:
Umm, ok, but I may want to write the output pdf directly to the USB, so I will remove host:ro but let's also remove ro from media.